test/test_torch.py

```diff
@@ -5301,6 +5300,197 @@ def add_neg_dim_tests():
 class TestTorchDeviceType(TestCase):
     exact_dtype = True
 
+    def _isclose_helper(self, tests, device, dtype, equal_nan, atol=1e-08, rtol=1e-05):
```

Comment:
do you want to remove test_allclose, as it is superceded by these tests?

Reply:
Off with its head!